bin/disco_purge_snapshots.py

```diff
@@ -40,6 +41,8 @@
 from disco_aws_automation.disco_logging import configure_logging
 
 OLD_IMAGE_DAYS = 100
+DEFAULT_KEEP_LAST = 5
+NOW = datetime.now(pytz.UTC)
```

Comment:
So that I can replace it in unit tests https://github.com/amplifylitco/asiaq/blob/582ff3c983782c920f75778cdbc74de3995c7115/test/unit/test_purge_snapshots.py#L66